src/state/comments.test.ts

```diff
@@ -21,6 +21,7 @@ test('Initial comments state empty', () => {
 test('New comment added to state', () => {
   const newComment: Comment = {
     contentpath: 'test_contentpath',
+    position: '',
```

Comment:
question: shouldn’t there be tests with a position defined to something?

Reply:
The position is only used externally, but I'm happy to put some text in there if you'd prefer!

Reply:
Oh indeed! I think what prompted me to ask was that it wasn’t very clear what type of value we should expect for position (beyond the fact it’s a string by default). But perhaps it doesn’t matter, if it’s only managed externally anyway? But in that case it might be more appropriate as an `unknown` type rather than string?